From e2590a57e8fd557aec5b8a64bcca9308a950414b Mon Sep 17 00:00:00 2001
From: Aaron Brodersen <aaron@abrodersen.com>
Date: Wed, 13 Jul 2016 12:12:50 -0500
Subject: [PATCH] Ignore unused return values

Credit to Hazerd and leafi for identifying the issue and resolution.
---
 src/pal/src/exception/seh.cpp                 | 2 +-
 tests/src/Common/Platform/platformdefines.cpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/pal/src/exception/seh.cpp b/src/pal/src/exception/seh.cpp
index 01de6e8..a89f800 100644
--- a/src/pal/src/exception/seh.cpp
+++ b/src/pal/src/exception/seh.cpp
@@ -233,7 +233,7 @@ SEHProcessException(PEXCEPTION_POINTERS pointers)
                     {
                         // The exception happened in the page right below the stack limit,
                         // so it is a stack overflow
-                        write(STDERR_FILENO, StackOverflowMessage, sizeof(StackOverflowMessage) - 1);
+                        (void)write(STDERR_FILENO, StackOverflowMessage, sizeof(StackOverflowMessage) - 1);
                         PROCAbort();
                     }
                 }
diff --git a/tests/src/Common/Platform/platformdefines.cpp b/tests/src/Common/Platform/platformdefines.cpp
index 4caead5..3a2cae2 100644
--- a/tests/src/Common/Platform/platformdefines.cpp
+++ b/tests/src/Common/Platform/platformdefines.cpp
@@ -277,7 +277,7 @@ DWORD TP_GetFullPathName(LPWSTR fileName, DWORD nBufferLength, LPWSTR lpBuffer)
 	return GetFullPathNameW(fileName, nBufferLength, lpBuffer, NULL);
 #else
 	char nativeFullPath[MAX_PATH];
-	realpath(HackyConvertToSTR(fileName), nativeFullPath);
+	(void)realpath(HackyConvertToSTR(fileName), nativeFullPath);
 	LPWSTR fullPathForCLR = HackyConvertToWSTR(nativeFullPath);
 	wcscpy_s(lpBuffer, MAX_PATH, fullPathForCLR);
 	return wcslen(lpBuffer);
-- 
2.9.0

